Tests/RealTests/RealTests.swift

```diff
@@ -59,6 +59,15 @@ internal extension Real where Self: BinaryFloatingPoint {
 
 final class ElementaryFunctionChecks: XCTestCase {
   
+  #if !os(macOS)
+  func testFloat16() {
+    if #available(iOS 14.0, watchOS 14.0, tvOS 7.0, *) {
```

Comment:
This pull builds and passes the tests except for this test, which doesn't compile on Android AArch64. [`if #available` is a runtime check that allows all unspecified platforms (see last section)](https://docs.swift.org/swift-book/LanguageGuide/ControlFlow.html), so it can't be used to avoid a compile-time feature like this.